server/tests/lib/nl/nl_page_config_next_test.py

```diff
@@ -0,0 +1,487 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration tests for NL Next chart generation."""
+
+import unittest
+from google.protobuf import text_format
+from parameterized import parameterized
+from typing import Dict
+from unittest.mock import patch
+
+from lib.nl import nl_utils, nl_utterance, nl_page_config_next
+from config.subject_page_pb2 import SubjectPageConfig
+from tests.lib.nl.test_utterance import PLACE_ONLY_UTTR, SIMPLE_UTTR, SIMPLE_WITH_SV_EXT_UTTR, \
+  SIMPLE_WITH_TOPIC_UTTR, COMPARISON_UTTR, CONTAINED_IN_UTTR, CORRELATION_UTTR, RANKING_UTTR
+
+PLACE_ONLY_CONFIG = """
```

Comment:
not necessary for this PR, is it better to have individual textproto test file for each of these?

Reply:
that would be nice, yes